apps/civicsignalblog/src/components/PageHeader/PageHeader.js

```diff
@@ -31,9 +32,14 @@ const PageHeader = React.forwardRef(function PageHeader(props, ref) {
         >
           {title}
         </RichTypography>
-        <RichTypography component="h2" variant="h2">
-          {subtitle}
-        </RichTypography>
+        <RichText
+          ref={ref}
```

Comment:
Why are we putting the component `ref` on this `RichText`? Shouldn't it be on parent/root component?

Reply:
Cant even recall when I kept this 🫠 ...removed ✅